core/src/main/java/io/questdb/cairo/map/FastMap.java

```diff
@@ -696,6 +697,15 @@ public void putTimestamp(long value) {
             putLong(value);
         }
 
+        @Override
+        public void putUuid(long lo, long hi) {
+            checkSize(UuidUtil.BYTES);
```

Comment:
UuidUtil.BYTES doesn't sound the same as Integer.BYTES. Perhaps we can rename UuidUtil to Uuid ?